plugins/boltz/swapmachine.go

```diff
@@ -56,13 +56,11 @@ func log(in FsmIn, msg string) {
 }
 
 func (s *SwapMachine) getSleepTime(in FsmIn) time.Duration {
-	if s.BoltzPlugin.ChainParams.Name == "mainnet" {
-		return 1 * time.Minute
-	}
-	return 5 * time.Second
+	return s.BoltzPlugin.getSleepTime()
 }
 
 func (s *SwapMachine) FsmNone(in FsmIn) FsmOut {
+	log(in, "Invalid state reached")
```

Comment:
shouldnt we set `IsFinished: true` ?

Reply:
I am not sure about the `log` already. `None` is not a state you could encounter during regular flow. So if you hit it, it doesn't mean swap has failed, it means the version of agent is somehow totally broken. So I guess you don't even want to transition to `None` state but just crash. The logic behind is that if you change agent binary to some newer build you might catch up from the last regular state and still bring the swap to a final state.